app/services/shopify_import/variants/create.rb

```diff
@@ -0,0 +1,56 @@
+module ShopifyImport
+  module Variants
+    class Create
+      def initialize(shopify_variant, spree_product)
+        @shopify_variant = shopify_variant
+        @spree_product = spree_product
+      end
+
+      def save!
+        Spree::Variant.transaction do
+          @spree_variant = build_spree_variant
+          add_option_values
+          @spree_variant.save!
+          set_stock_data
+        end
+      end
+
+      private
+
+      def build_spree_variant
+        Spree::Variant.new(parser.variant_attributes)
+      end
+
+      def add_option_values
+        @spree_variant.assign_attributes(option_value_ids: parser.option_value_ids)
+      end
+
+      def set_stock_data
```

Comment:
Should this gem also take into an account the spree's `track_inventory_levels` preference: https://github.com/spree/spree/blob/27673018bf689437d588035d21d3158625a0406d/core/app/models/spree/app_configuration.rb#L62
?